src/wallet/wallet.cpp

```diff
@@ -357,6 +357,35 @@ std::shared_ptr<CWallet> CreateWallet(WalletContext& context, const std::string&
     return wallet;
 }
 
+std::shared_ptr<CWallet> RestoreWallet(WalletContext& context, const std::string& backup_file, const std::string& wallet_name, std::optional<bool> load_on_start, const DatabaseOptions& options, DatabaseStatus& status, bilingual_str& error, std::vector<bilingual_str>& warnings)
```

Comment:
In 33763d02ef1011ae71143c5e6e313c72d9b790a6 "refactor: Move restorewallet() logic to the wallet section"

`options` does not need to be an argument to this function. The `DatabaseOptions` for loading a restored wallet are always going to be the same so you can make that object inside of this function.

Reply:
Done in 4807f73